vendor/licenses/grammar/language-imba.txt

```diff
@@ -0,0 +1,5 @@
+---
+type: grammar
+name: language-imba
+license: mit
+---
```

Comment:
This looks to have been manually added rather than pulled in from the grammar repo itself.

The grammar repo needs to explicitly include a license file which `script/add-grammar` will automatically pull in and assess.